site/src/pages/components/menu.explainer.mdx

```diff
@@ -370,96 +583,51 @@ We understand it would be helpful to have this feature be natively supported. If
-
-Sometimes, to reduce the amount of keyboard tabbing to bypass large navigation menus, a [**Disclosure Navigation Menu** pattern](https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-navigation/) or [**Menubar Navigation** pattern](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/examples/menubar-navigation/) may be used.  Each have their own pros and cons.
-
-We are evaluating the possibility to include **Navigation Menubar** pattern in our proposal.  We need to carefully evaluate if this is a valuable addition, as there are long standing UX concerns with using menubars in this way, and if such a pattern is introduced, we want to make sure it addresses the concerns of all users.
-
-**Questions**
-
-* Should we reuse the [`<nav>` element](https://html.spec.whatwg.org/multipage/#the-nav-element) if we were to support a "disclosure" or "menubar" navigation pattern?
-* Instead of using `<menubar>`, should we have a new `<navmenubar>` element?
-* Should we support anchor elements inside `<menuitem>`s?
-    * No, `<menuitem>` should only wrap non-interactive elements.
-* Should we support href attribute on `<menuitem>`s?
-    * Maybe, if we commit to the "navigation menu" use case.
+### How does this proposal relate to navigation menus?
 
-**Questions about VoiceOver on navigation menubar**
+Navigation menus allow users to access different pages or sub-pages of a web application. They
+commonly appear in a few different forms:
 
-1. I tried Voiceover with on the chrome menus themselves and commands affecting the current context like copy/past aren't obviously distinguishable from commands that are more navigation like (e.g. settings...). Why is that uniquely important for the web?
-2. I tried Voiceover on google docs and tab navigation for the menus doesn't seem to work very well but when I am able to tab to Help => Privacy Policy (opens a new tab), it's not announced as a link.
+ - The `<nav>` element, which implicitly exposes the ARIA landmark
+   [`navigation`](https://w3c.github.io/aria/#navigation) role.
+    - Since hyperlinks and buttons are commonly children of this sort of navigation menu, authors
+      generally do not need to declare specific ARIA roles to be accessible, and users can rely on
+      standard <kbd>Tab</kbd> keey navigation to reach each interactive navigation element.
+ - The abused [Navigation Menubar Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/examples/menubar-navigation/)
+    - These generally have poor accessibility characteristics, and are not recommended.
+ - The [Disclosure Navigation Menu Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/examples/menubar-navigation/)
```

Comment:
This link is identical to the one just above, which is confusing since one says "this is bad" and the other says "this is better".